generators/server/templates/src/main/java/package/config/SecurityConfiguration_imperative.java.ejs

```diff
@@ -180,39 +180,39 @@ public class SecurityConfiguration {
                 // prettier-ignore
                 authz
 <%_ if (!skipClient) { _%>
-                    .requestMatchers(mvc.pattern("/"), mvc.pattern("/index.html"), mvc.pattern("/*.js"), mvc.pattern("/*.map"), mvc.pattern("/*.css")).permitAll()
-                    .requestMatchers(mvc.pattern("/*.ico"), mvc.pattern("/*.png"), mvc.pattern("/*.svg"), mvc.pattern("/*.webapp")).permitAll()
-                    .requestMatchers(mvc.pattern("/app/**")).permitAll()
-                    .requestMatchers(mvc.pattern("/i18n/**")).permitAll()
-                    .requestMatchers(mvc.pattern("/content/**")).permitAll()
-                    .requestMatchers(mvc.pattern("/swagger-ui/**")).permitAll()
+                    .requestMatchers(mvc.pattern("/"), AntPathRequestMatcher.antMatcher("/*.js"), AntPathRequestMatcher.antMatcher("/*.map"), AntPathRequestMatcher.antMatcher("/*.css")).permitAll()
+                    .requestMatchers(AntPathRequestMatcher.antMatcher("/*.ico"), AntPathRequestMatcher.antMatcher("/*.png"), AntPathRequestMatcher.antMatcher("/*.svg"), AntPathRequestMatcher.antMatcher("/*.webapp")).permitAll()
+                    .requestMatchers(AntPathRequestMatcher.antMatcher("/app/**")).permitAll()
+                    .requestMatchers(AntPathRequestMatcher.antMatcher("/i18n/**")).permitAll()
+                    .requestMatchers(AntPathRequestMatcher.antMatcher("/content/**")).permitAll()
+                    .requestMatchers(AntPathRequestMatcher.antMatcher("/swagger-ui/**")).permitAll()
```

Comment:
Personally, I used a static import to make it look nicer and more concise but it's OK as it is ;)

~~~java
import static org.springframework.security.web.util.matcher.AntPathRequestMatcher.antMatcher;

                   .requestMatchers(antMatcher("/swagger-ui/**")).permitAll()
~~~